docs/reference/commandline/dockerd.md

```diff
@@ -101,6 +102,82 @@ type `dockerd`.
 
 To run the daemon with debug output, use `dockerd -D`.
 
+> **Note**: In Docker 1.13 and higher, enable experimental features by starting
+> `dockerd` with the `--experimental` flag. In earlier Docker versions, a
+> different build was required to enable experimental features.Usage:	dockerd COMMAND
+
+A self-sufficient runtime for containers.
+
+Options:
```

Comment:
@mstanleyjones hum I think @AkihiroSuda means the `[…] features.Usage:	dockerd COMMAND` and below 😅 which seems like a copy/paste problem indeed 👼 as the output of `--help` is repeated here for, I think, no reason ?